examples/medplum-nextjs-demo/app/root.tsx

```diff
@@ -14,6 +14,9 @@ const medplum = new MedplumClient({
 
   // Use Next.js fetch
   fetch: (url: string, options?: any) => fetch(url, options),
+
+  // Recommend using cache for React performance
+  cacheTime: 10000,
```

Comment:
The check for `typeof window === 'undefined'` does not work in Next.js due to the Next.js build process, despite this file including `'use client';`